api_proxy.yaml

```diff
@@ -271,18 +358,34 @@ definitions:
     $ref: "https://raw.githubusercontent.com/teamdigitale/digital-citizenship-functions/v0.40.0/api/definitions.yaml#/OrganizationName"
   PaginationResponse:
     $ref: "https://raw.githubusercontent.com/teamdigitale/digital-citizenship-functions/v0.40.0/api/definitions.yaml#/PaginationResponse"
+  PaymentActivationsGetResponse:
+    $ref: "https://raw.githubusercontent.com/teamdigitale/italia-pagopa-proxy/skeleton_with_soap_server/api_pagopa.yaml#/definitions/PaymentActivationsGetResponse"
```

Comment:
we should wait to merge this PR until https://github.com/teamdigitale/italia-pagopa-proxy/pull/46 gets merged